config/CLIConfiguration.ts

```diff
@@ -228,23 +237,53 @@ class _CLIConfiguration {
   }
 
   getDefaultAccount(): string | number | null {
-    return this.config && this.config.defaultAccount
-      ? this.config.defaultAccount
-      : null;
+    return (
+      this.getResolvedDefaultAccountForCWD() ||
```

Comment:
Should `getResolvedDefaultAccountForCWD` return the configured account name for the account instead of the accountId? It would make this more consistent with the previous return type for `getDefaultAccount`, which technically can support the account ID but typically is the account name string.

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/228/commits/175f2106db5c2a7d37a17330d49148fa274a06a1